<launch>
  <rosparam>
    families: ["Lily"]
    names: [
        "L1_J1_base", "L1_J2_shoulder", "L1_J3_elbow",
        "L2_J1_base", "L2_J2_shoulder", "L2_J3_elbow",
        "L3_J1_base", "L3_J2_shoulder", "L3_J3_elbow",
        "L4_J1_base", "L4_J2_shoulder", "L4_J3_elbow",
        "L5_J1_base", "L5_J2_shoulder", "L5_J3_elbow",
        "L6_J1_base", "L6_J2_shoulder", "L6_J3_elbow",
    ]
    gains_package: "hebi_cpp_api_examples"
    gains_file: "data/hexapod/gains18.xml"
    hrdf_package: "hebi_cpp_api_examples"
    hrdf_file: "data/hexapod/lily.hrdf"
    message_timeout: "0.5"
  </rosparam>
  <node name="group_node" pkg="hebi_cpp_api_examples" type="group_node" output="screen"/>
</launch>
